mapbox/services/directions.py

```diff
@@ -16,7 +19,8 @@ class Directions(Service):
 
     @property
     def baseuri(self):
-        return 'https://{0}/v4/directions'.format(self.host)
+        return 'https://{0}/{2}/{1}'.format(
```

Comment:
@sgillies , can property `baseuri` be inherited from base service class here?

Reply:
Not for this class because the Directions API has the form `/{version}/directions` instead of the more standard `/directions/{version}`. It would be great if our API URIs followed one single pattern, but they don't.

Reply:
Oops, okay.